packages/cloud-cognitive/src/components/Datagrid/Extensions/ClickableRow/ClickableRow.stories.js

```diff
@@ -7,7 +7,7 @@
  */
 
 import React, { useState } from 'react';
-import { Edit16, TrashCan16 } from '@carbon/icons-react';
+import { Edit, TrashCan } from '@carbon/icons-react';
```

Comment:
Although this will technically work because `@carbon/icons-react` exists somewhere in the dependency tree, we can't guarantee it since it's not listed in `package.json`. We can by instead consuming icons via `@carbon/react`:

```suggestion
import { Edit, TrashCan } from '@carbon/react/icons';
```

Reply:
Oh thanks for pointing this out! I imagine this change will need to be made across several other components, I'll open an issue for us to change this.

Reply:
https://github.com/carbon-design-system/ibm-cloud-cognitive/issues/2319

Reply:
Updated in https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/2320